tracim/tracim/lib/webdav/__init__.py

```diff
@@ -5,11 +5,14 @@
 from wsgidav import compat
 
 from tracim.lib.content import ContentApi
+from tracim.lib.utils import SameValueError
+from tracim.lib.base import logger
 from tracim.model import new_revision
 from tracim.model.data import ActionDescription
 from tracim.model.data import ContentType
 from tracim.model.data import Content
 from tracim.model.data import Workspace
+from wsgidav.dav_error import DAVError, HTTP_FORBIDDEN, HTTP_NOT_MODIFIED
```

Comment:
`import` code style to follow: "Imports" chapter in http://algoo.trac.im/workspaces/11/folders/1194/pages/2043

Reply:
HTTP_NOT_MODIFIED is not used anymore, i drop it.